README.md

```diff
@@ -1,2 +1,82 @@
-# orleans-templates
-Templates for build Microsoft Orleans based solutions
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Orleans.Templates.svg?style=flat)](http://www.nuget.org/profiles/Orleans)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/orleans?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+# Overview
+
+This project contains the Project and Item templates to make it easier to getting started with [Orleans](https://github.com/dotnet/orleans).
+
+The templates are leveraging the new [dotnet templating engine](https://github.com/dotnet/templating).
+
+# Provided templates
+
+
+# Installation
```

Comment:
Please add a section on how to build (as well as a build script). I was not able to build from command line (both MSBUILD nor dotnet) nor VS 2017.4.1
This is the output from `dotnet build`:

```
> dotnet build
Microsoft (R) Build Engine version 15.4.8.50001 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Users\xxx\.nuget\packages\microsoft.vssdk.buildtools\15.5.72\tools\VSSDK\Microsoft.VsSDK.targets(82,5): error MSB4062: The "CompareCommonBuildTaskVersion" task could not be loaded from the assembly C:\Users\xxx\.nuget\packages\microsoft.vssdk.buildtools\15.5.72\tools\VSSDK\Microsoft.VisualStudio.Sdk.BuildTasks.dll. Could not load file or assembly 'System.IO.Packaging, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [D:\projects\orleans-templates\src\Orleans.Templates\Orleans.Templates.csproj]

Build FAILED.

C:\Users\xxx\.nuget\packages\microsoft.vssdk.buildtools\15.5.72\tools\VSSDK\Microsoft.VsSDK.targets(82,5): error MSB4062: The "CompareCommonBuildTaskVersion" task could not be loaded from the assembly C:\Users\xxx\.nuget\packages\microsoft.vssdk.buildtools\15.5.72\tools\VSSDK\Microsoft.VisualStudio.Sdk.BuildTasks.dll. Could not load file or assembly 'System.IO.Packaging, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [D:\projects\orleans-templates\src\Orleans.Templates\Orleans.Templates.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.31
```